account_invoice_pricelist/README.rst

```diff
@@ -1,62 +1,82 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
You can keep the original README.rst file as it was already done. No need to replace it.
As far as I understood during this code sprint, the readme will be overwritten automatically by a bot during the night, after this PR is merged. It means that in the meanwhile the original README.rst is still needed.